### PR TITLE
[ranges] Harmonize index entries.

### LINKED ISSUE #3102: [ranges] Switch to using \indexlibrarymember throughout

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #3102.

### CODE REVIEW

File: source/ranges.tex [hunk 1]
```diff
@@ -4063,6 +4065,7 @@
 }
 \end{codeblock}
 
+\indexlibrary{\idxcode{take_while_view}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex [hunk 2]
```diff
@@ -4117,6 +4124,7 @@
 }
 \end{codeblock}
 
+\indexlibrary{\idxcode{take_while_view::sentinel}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex [hunk 3]
```diff
@@ -4127,6 +4135,7 @@
 Initializes \tcode{end_} with \tcode{end} and \tcode{pred_} with \tcode{pred}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{take_while_view::sentinel}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex [hunk 4]
```diff
@@ -5860,6 +5877,7 @@
 }
 \end{codeblock}
 
+\indexlibrary{\idxcode{basic_istream_view::iterator}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex [hunk 5]
```diff
@@ -6137,6 +6160,7 @@
 }
 \end{codeblock}
 
+\indexlibrary{\idxcode{elements_view::iterator}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex [hunk 6]
```diff
@@ -6147,6 +6171,7 @@
 Initializes \tcode{current_} with \tcode{current}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{elements_view::iterator}!constructor}%
```
Comment (zygoloid) [inline source/ranges.tex]: `indexlibraryctor`

File: source/ranges.tex (hunk 1 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 3 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 4 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 5 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 6 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 2 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

**Comment by jensmaurer** [comment]
Rebased and fixed applied.

**Pull request merged**